### ISSUE TITLE
Low performance when open xlsx file with many formulas

### ISSUE THREAD

**Issue by fz0000** [ONLYOFFICE/DocumentServer#2683]
I have an Excel file which has many fomulas.
It would cost 1-2 seconds to select an item from the list. I tried on both my local server and the official DocSpace online and there's no difference.

https://github.com/ONLYOFFICE/CommunityServer/assets/5061311/d94ddaaf-37ac-4ff5-9555-0e5e0ab3218b

Is it expected?

The test file is attached.
[performance-test.xlsx](https://github.com/ONLYOFFICE/CommunityServer/files/14823375/performance-test.xlsx)

**Comment by rsnegirev** [comment]
Hi @fz0000 

I can confirm, this is the bug number 49009. When some fixes will be done we'll text you here in this issue.

Thank you for the report!

**Comment by by-evgenii** [comment]
XLSX file. ~1500  rows, 5 columns with formulas, pretty simple ones. In Excel, it works blazing fast. 
In the Only  - copying of the column ~20 seconds. 

Formula types: 
- A2 = A1+1;
- C2 = countif(D2, "*STR*");
- E2= IF(ISEMPTY(B2);"";MONTH(B2))
- F2 = IF(ISEMPTY(B2);"";WEEK(B2))
- IG = IF(ISEMPTY(B2);"";DAY(B2))

core i5 8300h, 16Gb RAM, Windows 11

**Comment by fz0000** [comment]
Yes, I attempted to copy from one column to another, and it hangs.
But it seems that the dropdown list is working fast now. So this bug might still be getting fixed.

Verified on a clean installed Workspace 12.7.1.1942 (with DocumentServer 9.2.0).
